src/common/helper.ts

```diff
@@ -182,6 +182,115 @@ function evaluationString(fun: Function | string, ...args: unknown[]): string {
   return `(${fun})(${args.map(serializeArgument).join(',')})`;
 }
 
+function pageBindingInitString(type: string, name: string): string {
+  return evaluationString(addPageBinding, type, name);
+
+  function addPageBinding(type: string, bindingName: string): void {
+    /* Cast window to any here as we're about to add properties to it
+     * via win[bindingName] which TypeScript doesn't like.
+     */
+    const win = window as any;
+    const binding = win[bindingName];
+
+    win[bindingName] = (...args: unknown[]): Promise<unknown> => {
+      const me = window[bindingName];
+      let callbacks = me['callbacks'];
```

Comment:
In general, use dot notation except when you need dynamic property names

`me['callbacks']` -> `me.callbacks`

(and so on elsewhere in this file)

Reply:
Thanks, done.